hw/amdc_revd.bd

```diff
@@ -1180,7 +801,10 @@
                 "value": "MIO 16 .. 27"
               },
               "PCW_ENET0_GRP_MDIO_ENABLE": {
-                "value": "0"
+                "value": "1"
+              },
+              "PCW_ENET0_GRP_MDIO_IO": {
+                "value": "MIO 52 .. 53"

```

Comment:
Per our discussion, this is the only real change here... Interesting to see all the other Vivado bd file changes... Silly that it changes so much without any functional changes.